email/src/main/java/edu/jhuapl/dorset/demos/EmailClient.java

```diff
@@ -0,0 +1,102 @@
+package edu.jhuapl.dorset.demos;
+
+import java.util.Scanner;
+
+import javax.mail.Message;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+import edu.jhuapl.dorset.Application;
+import edu.jhuapl.dorset.Request;
+import edu.jhuapl.dorset.Response;
+import edu.jhuapl.dorset.agents.Agent;
+import edu.jhuapl.dorset.agents.DateTimeAgent;
+import edu.jhuapl.dorset.routing.Router;
+import edu.jhuapl.dorset.routing.SingleAgentRouter;
+
+public class EmailClient {
+
+    private static final Logger logger = LoggerFactory.getLogger(EmailClient.class);
+    private static EmailManager manager;
+
```

Comment:
Let's not have manager and the get methods and processMessage() be static. Instead, construct an EmailClient object in the main method so it looks like this:

```java
EmailClient client = new EmailClient(new EmailManager(config));
client.run();
client.close();
```
Then the run method implements your loop and the close method cleans up the manager.